build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -19,160 +19,115 @@ public static class PublishTargets
 
         private static DebRepoPublisher DebRepoPublisherTool { get; set; }
 
+        private static VersionRepoUpdater VersionUpdater { get; set; }
+
         private static string Channel { get; set; }
 
+        private static string Pointer { get; set; }
+
         private static string CliVersion { get; set; }
 
         private static string CliNuGetVersion { get; set; }
 
-        private static string SharedFrameworkNugetVersion { get; set; }
-
         [Target]
         public static BuildTargetResult InitPublish(BuildTargetContext c)
         {
             AzurePublisherTool = new AzurePublisher();
             DebRepoPublisherTool = new DebRepoPublisher(Dirs.Packages);
+            VersionUpdater = new VersionRepoUpdater(EnvVars.EnsureVariable("GITHUB_PASSWORD"));
```

Comment:
This is probably going to fail on Jenkins and everyone's machines since this EnvVar isn't set.  It is only set on VSO.